astropy/vo/samp/tests/test_standard_profile.py

```diff
@@ -266,7 +265,6 @@ def hub_init_kwargs(self):
                }
 
 
-@pytest.mark.xfail
```

Comment:
Before we merge this PR we might want to tag them as ``xfail`` again because they do have sporadic failures. Or maybe we should flag them ``remote_data`` because they have the same kind of flakyness. But let's keep this removed for now to see if Travis passes.